tools/cli/domainCommands.go

```diff
@@ -854,6 +856,70 @@ func (d *domainCLIImpl) migrationDomainWorkFlowCheck(c *cli.Context) DomainMigra
 	}
 }
 
+func (d *domainCLIImpl) searchAttributesChecker(c *cli.Context) DomainMigrationRow {
+	ctx, cancel := newContext(c)
+	defer cancel()
+
+	// getting user provided search attributes
+	searchAttributes := c.StringSlice(FlagSearchAttribute)
+	if len(searchAttributes) == 0 {
+		ErrorAndExit("Please provide search attributes.", nil)
```

Comment:
it's possible that they don't use search attributes. In that case, we can return pass.
```suggestion
		return DomainMigrationRow{
		  ValidationCheck:  "Search Attributes Check",
		  ValidationResult: true,
		}
```